frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/utils.ts

```diff
@@ -1,9 +1,31 @@
 import * as _ from 'lodash';
-import { TektonParam } from '../../../types';
+import { getResourceModelFromTaskKind } from '../../../utils/pipeline-augment';
+import { PipelineKind, PipelineTask, TektonParam } from '../../../types';
+import { ResourceModelLink } from '../resource-overview/DynamicResourceLinkList';
 
 export const removeEmptyDefaultFromPipelineParams = (parameters: TektonParam[]): TektonParam[] =>
   _.map(
     parameters,
     (parameter) =>
       _.omit(parameter, _.isEmpty(parameter.default) ? ['default'] : []) as TektonParam,
   );
+
+export type PipelineTaskLinks = {
+  taskLinks: ResourceModelLink[];
+  finallyTaskLinks?: ResourceModelLink[];
```

Comment:
You are always return finallyTaskLink :thinking: shouldn't it be
```suggestion
  finallyTaskLinks: ResourceModelLink[];
```